docs/containers-registries.conf.5.md

```diff
@@ -236,6 +236,12 @@ location = "example-mirror-0.local/mirror-for-foo"
 [[registry.mirror]]
 location = "example-mirror-1.local/mirrors/foo"
 insecure = true
+
+[[registry]]
+location = "registry.com"
+
+[[registry.mirror]]
+location = "mirror.registry.com"
 ```
 Given the above, a pull of `example.com/foo/image:latest` will try:
     1. `example-mirror-0.local/mirror-for-foo/image:latest`
```

Comment:
Not part of this change, but it looks like there's an extra leading space on this line.

Reply:
Let’s fix that: https://github.com/containers/image/pull/1527 (_after_ this PR).